its/projects/TwoDrivesThreeProjects/setup.bat

```diff
@@ -0,0 +1 @@
+SUBST Y: .\DriveY\
```

Comment:
this fails on the VMs that run VS 2022 jobs

https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=52300&view=logs&jobId=73f8605c-9176-5092-6bed-e0c8f7a97c23&j=73f8605c-9176-5092-6bed-e0c8f7a97c23&t=688cb1cf-c272-5f81-3f26-fa784ad4c6a3